### ISSUE TITLE
v2.1.505+dependencies tag is wrong

### ISSUE THREAD

**Issue by vellozzi** [dotnet/cli#10969]
If you look at the [v2.1.505+dependencies](https://github.com/dotnet/cli/commits/v2.1.505%2Bdependencies) tag the tagged commit is [ad523ce44af2876afb3eddbf5a71fdf294ac2bd7](https://github.com/dotnet/cli/commit/ad523ce44af2876afb3eddbf5a71fdf294ac2bd7).

But if you look at the [Release/2.1.5xx branch](https://github.com/dotnet/cli/commits/release/2.1.5xx) commit [35fbd6d033612a55ab4a1fcdfbd3ab4c36e8b49e](https://github.com/dotnet/cli/commit/35fbd6d033612a55ab4a1fcdfbd3ab4c36e8b49e) looks like it's the one that should be tagged as v2.1.505+dependencies.

**Comment by livarcocc** [comment]
No, the tag is correct. Without that commit when you build the CLI locally, the implicit versions used in the recently build CLI won't be correct. They will be old and match your stage0 CLI's implicit version.

This was not a concern in our ProdCon build CLI because over there the implicit versions were passed in through msbuild global properties.

**Comment by vellozzi** [comment]
Does that mean that the previous tag is wrong then? The [2.1.504+dependencies](https://github.com/dotnet/cli/commits/v2.1.504%2Bdependencies) tag includes the version updates in [this commit](https://github.com/dotnet/cli/commit/03be46649c7a9e41fd61d5907cf31dd72fd4bcd2)

**Comment by vellozzi** [comment]
So you're saying that [this commit](https://github.com/dotnet/cli/commit/35fbd6d033612a55ab4a1fcdfbd3ab4c36e8b49e) that updates some of the dependencies to 2.1.9 doesn't belong in the 2.1.505+dependencies tag even though 2.1.505 ships with 2.1.9 of the runtime?

**Comment by vellozzi** [comment]
I built with the 2.1.505+dependencies tag and it's either picking up Microsoft.NETCore.App 2.1.8 instead of 2.1.9 or it's putting it in a directory called 2.1.8.

**Comment by livarcocc** [comment]
The 2.1.504 is correct. However, during the 2.1.505 servicing we found an issue with ProdCon, fixed that issue by using msbuild properties which worked on prodcon because they were global and then found out that they broke local builds, so we pushed that extra commit.

However, I see now that the +dependencies tag did not include the update versions commit. Great catch and my bad on that. Looking into fixing now. Sorry about this.
